cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -292,3 +265,44 @@ func getEtcdCommand(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.A
 	command = append(command, kubeadmutil.BuildArgumentListFromMap(defaultArguments, cfg.Etcd.Local.ExtraArgs)...)
 	return command
 }
+
+func getAndWriteEtcdStaticPod(manifestDir string, patchesDir string, cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, nodeName string, initialCluster []etcdutil.Member, isDryRun bool) error {
+	// gets etcd StaticPodSpec, actualized for the current InitConfiguration and the new list of etcd members
```

Comment:
```suggestion
	// gets etcd StaticPodSpec, actualized for the current ClusterConfiguration and the new list of etcd members
```

an opportunity to change Init->Cluster. this also happens during "join".

Reply:
Done!